guidance/src/guidance_state_machine.cpp

```diff
@@ -83,34 +85,29 @@ namespace guidance
         }
     }
 
-    void GuidanceStateMachine::onVehicleStatus(const autoware_msgs::VehicleStatusConstPtr& msg)
+    void GuidanceStateMachine::onVehicleStatus(autoware_msgs::msg::VehicleStatus::UniquePtr msg)
     {
         current_velocity_ = msg->speed * 0.277777; // Convert kilometers per hour to meters per second. Rounded down so that it comes under epsilon for parking check
         if (current_guidance_state_ == State::ENTER_PARK)
         {
             // '3' indicates vehicle gearshift is currently set to PARK
-            if(msg->current_gear.gear== autoware_msgs::Gear::PARK)
+            if(msg->current_gear.gear== autoware_msgs::msg::Gear::PARK)
             {
                 onGuidanceSignal(Signal::OVERRIDE); // Required for ENTER_PARK -> INACTIVE
 
-                if(operational_drivers_){
-                    onGuidanceSignal(Signal::INITIALIZED); 
-                }
+                onGuidanceSignal(Signal::INITIALIZED); // Required for INACTIVE -> DRIVERS_READY
```

Comment:
Is there a change elsewhere that explains why this check is no longer needed?

Reply:
I spoke with Mike about the check for all drivers being ready on /system_alert, and he mentioned that health_monitor will be removed once our system has been migrated to ROS2. Instead, the guidance subsystem will have required nodes (including required drivers), and if any of these shutdown, then I believe all guidance subsystem nodes will shutdown. Because of this, the check for the `operational_drivers_` flag is no longer needed. I did notice though that I left that boolean flag in guidance_state_machine.hpp, and will remove it now.